tests/fsharpqa/Source/Conformance/InferenceProcedures/NameResolution/RequireQualifiedAccess/OnRecordVsUnion.fs

```diff
@@ -0,0 +1,11 @@
+// #Conformance #TypeInference #Attributes 
+// Verify the RequireQualifiedAccess attribute works on unions
+
+module A =
+    [<RequireQualifiedAccess>]
+    type U = | C
+
+    type C() =
+        static member M() = ()
+
+let x = A.C.M()
```

Comment:
https://github.com/Microsoft/visualfsharp/pull/1512/commits/d8073c2aaf118be1647d198ab42197b9c9162d2c#diff-d59739c861a71a0c665896efcd95507dR4

question is: is that correct